Purchases/Public/Package.swift

```diff
@@ -71,36 +71,27 @@ private extension PackageType {
 
     @objc public let identifier: String
     @objc public let packageType: PackageType
-    @objc public let product: SKProduct
+    @objc public let productWrapper: ProductWrapper
```

Comment:
What do you think about naming this just `product`? Do developers care about it being a wrapper or are they normally just going to access the properties in the `ProductWrapper` directly? The equivalent in Android for this is `ProductDetails`.

Reply:
yeah, i think i like that too. to be honest, even ProductDetails confused me a little on android.

Reply:
Yeah, we have the problem you're mentioning with `Product` in Android but with `Package`, since it there's an `android.os.Package`, and to be fair, it's quite annoying and I wish we hadn't named it `Package` lol.